scripts/validate_snqi_scripts.py

```
@@ -0,0 +1,217 @@
+        sys.path.insert(0, str(scripts_dir))
+        from recompute_snqi_weights import SNQIWeightRecomputer
+
+        recomputer = SNQIWeightRecomputer(episodes, baseline_stats)
+
+        # Test default weights
+        default_weights = recomputer.default_weights()
+        print("✓ Default weights generation: PASS")
+
+        # Test SNQI computation
+        for i, episode in enumerate(episodes):
+            score = recomputer.compute_snqi(episode["metrics"], default_weights)
```

Pylint (3.3.7)

[error] Instance of 'SNQIWeightRecomputer' has no 'compute_snqi' member

(E1101)